pulsar-client-cpp/include/pulsar/ConsumerConfiguration.h

```diff
@@ -378,6 +381,20 @@ class PULSAR_PUBLIC ConsumerConfiguration {
      */
     InitialPosition getSubscriptionInitialPosition() const;
 
+    /**
+     * Set batch receive policy.
+     *
+     * @param batchReceivePolicy the default is xxx
```

Comment:
The documents are really bad. What's "xxx"?

If you are not good at writing documents, you can add the `doc-required` label and open an issue for the documents part. /cc @Anonymitaet

Reply:
Sorry, i missed it here.